installer/META-INF/com/google/android/update-binary

```diff
@@ -32,7 +32,9 @@ extract() {
 }
 
 print "#########################################"
-print "#  TWRP installer for Moto G7 (river)   #"
+print "#   TWRP installer for Fairphone FP3    #"
+print "#             by @k4y0z                 #"
+print "#  based on TWRP installer for Moto G7  #"
 print "#            by @SyberHexen             #"
```

Comment:
> Also I don't think this script was actually written by @SyberHexen.
> As the header indicates:
> 
> ```
> # LazyFlasher installer backend (for helper functions only)
> # Heavily modified by Dees_Troy and osm0sis
> ```

You're right originally kanged from the official oneplus guacamole twrp tree.
Which this only is useful if you're building Unofficial twrp.

Reply:
Thanks for joining the discussion @SyberHexen.
Yeah, only unofficial TWRP for now (release was 2 days ago...)
So I guess you'd be fine with removing your name from the credits in the script?
Seems like most maintainers who use that script, just add their own name.

BTW the script currently breaks boot if magisk was previously installed (probably because magisk patches skip_initramfs out).
Could probably patch that back in with magiskpatch

Reply:
> Thanks for joining the discussion @SyberHexen.
> Yeah, only unofficial TWRP for now (release was 2 days ago...)
> So I guess you'd be fine with removing your name from the credits in the script?
> Seems like most maintainers who use that script, just add their own name.
> 
> BTW the script currently breaks boot if magisk was previously installed (probably because magisk patches skip_initramfs out).
> Could probably patch that back in with magiskpatch

That's fine, Installer tag i felt was more for distribution credits of an un-official installer
Taken from a recent production build of twrp installer for ASUS Rog Phone 2 you can see the many differences here in a newer updater-binary
https://del.dog/yttoraphol.txt

Reply:
Ha, looks like it does exactly what I was suggesting:

```
  # kernel string want_initramfs -> skip_initramfs (Magisk)
  $tool hexpatch kernel 77616E745F696E697472616D6673 736B69705F696E697472616D6673;
  # kernel string trip_initramfs -> skip_initramfs (SuperSU)
  $tool hexpatch kernel 747269705F696E697472616D6673 736B69705F696E697472616D6673;
```